daemon/execdriver/lxc/driver.go

```diff
@@ -87,6 +87,12 @@ func (d *driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 		"-n", c.ID,
 		"-f", configPath,
 	}
+
+	// From lxc>=1.1 the default behavior is to daemonize containers after start
+	if d.version() >= "1.1" {
```

Comment:
i'm also not fond of comparing version. However the check here will *probably* always work because the version we are comparing to is "1.1.x". But may be a check to see if `foreground` option is available via `lxc-start --help` is better ?